scipp_widgets/inputs.py

```diff
@@ -6,6 +6,8 @@
 from scipp_widgets.validators import ScippObjectValidator, AttrValidator
 from typing import Any, Sequence, Callable
 from abc import ABC, abstractmethod
+from ipyfilechooser import FileChooser
```

Comment:
Can we import this only when `FileChooser` is used, so this becomes an optional dependency?